pages/about.html

```diff
@@ -1,12 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/static/styles.css" />
+
+    <title>About Yonese</title>
+    
 </head>
 <body>
-  
+
+<!-- Navbar Section -->
+<nav class="navbar">
```

Comment:
You could put this nav element inside a header element in the body to add more semantic meaning, like:

```HTML 
<body>
    <header>
        <nav>